cli/test/smokehouse/config/exclusions.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
nit: maybe rename the directory too to not overload `config`?

Reply:
> It's not the end of the world to remember to go remove it from the exclusions list at that point

I added a console line that lists excluded tests when a requested test is mismatched. So in this scenario, it would print:

```
Smoketests not found for: byte-efficiency
Check test exclusions (byte-efficiency)
```

This should help reminding the dev running smoke why a test can't be found. If this and the config edit in practice turns out inconvenient, we can add the `--ignore-exclusions` that @connorjclark initially suggested.